site/app/templates/admin/RainbowCustomization.twig

```diff
@@ -155,6 +155,65 @@
+                <input type="hidden" name="option" value="-1" /> <!-- I'm not confident I know what this does -->
+                <div class="option">
+                    <label for="manual_grading_user_id">
+                        <span>USER ID:</span>
+                        <input class="option-input" type="text" name="manual_grading_user_id" id="manual_grading_user_id">
+                    </label>
+                    <label for="manual_grading_grade">
+                        <span>Final Grade:</span>
+                        <select name="manual_grading_grade" id="manual_grading_grade">
+                            <option disabled selected value> -- select an option -- </option>
+                            <option value="A">A</option>
+                            <option value="A-">A-</option>
+                            <option value="B+">B+</option>
+                            <option value="B">B</option>
+                            <option value="B-">B-</option>
+                            <option value="C+">C+</option>
+                            <option value="C">C</option>
+                            <option value="C-">C-</option>
+                            <option value="D+">D+</option>
+                            <option value="D">D</option>
+                            <option value="F">F</option>
+                        </select>
+                    </label>
+                    <label for="manual_grading_note">
+                        <span>Note:</span>
+                        <input class="option-input" type="text" name="manual_grading_note" id="manual_grading_note">
+                    </label>
+                    <button type="submit" class="btn btn-primary" onclick="addToManualGradingTable()" aria-label="submit">
+                        <i class="fas fa-plus"></i>
+                    </button>
+                    {# This is a data table #}
+                    <table id="manual-grading-table" class="table table-striped table-bordered persist-area mobile-table">
+                        <thead>
+                            <tr style="text-align:left">
+                                <th>USER ID</th>
+                                <th>Final Grade</th>
+                                <th>Note</th>
+                                <th>Delete</th>
+                            </tr>
+                        </thead>
+                        <tbody id="manual-grading-table-body" style="text-align:left">
+                            {% for entry in manual_grade %}
+                                <tr>
+                                    <td>{{ entry.user }}</td>
+                                    <td>{{ entry.grade }}</td>
+                                    <td>{{ entry.note }}</td>
+                                    <td><a onclick="deleteRow(this)"><i class="fas fa-trash"></i></a></td>
```

Comment:
deleting a row does not seem to prompt the user to save changes (it stills says "no changes detected")

Reply:
Fixed in [0d4e524](https://github.com/Submitty/Submitty/pull/10653/commits/0d4e52443ded2939cf2dcbf4021d557ef594b416)